includes/admin/functions-post.php

```diff
@@ -258,11 +258,22 @@ function wpas_save_ticket( $post_id ) {
 	/* If this was a ticket update, we need to know where to go now... */
 	if ( '' !== $original_status ) {
 
+		$gt_post = null;
+		
 		/* Go back to the tickets list */
-		if ( isset( $_POST['wpas_back_to_list'] ) && true === boolval( $_POST['wpas_back_to_list'] ) || isset( $_POST['where_after'] ) && 'back_to_list' === $_POST['where_after'] ) {
+		if ( ( isset( $_POST['wpas_back_to_list'] ) && true === boolval( $_POST['wpas_back_to_list'] ) ) || ( isset( $_POST['where_after'] ) && 'back_to_list' === $_POST['where_after'] ) ) {
```

Comment:
With this many `if` statements, better use `switch()` instead.

```
if ( isset( $_POST['wpas_back_to_list'] ) && true === boolval( $_POST['wpas_back_to_list'] ) ) {
    // Action
} elseif ( isset( $_POST['where_after'] ) ) {
    switch( isset( $_POST['where_after'] ) ) {
        // Add cases
    }
}
```

Also, we should sanitize the input by using [`filter_input()`](http://php.net/manual/en/function.filter-input.php).

`$_SESSION['wpas_redirect']` should definitely not be used (we need to leverage the `WPAS_Session` class instead). This is maybe something I should deal with afterward. Let me know if you'd like me to make the changes after this PR is merged.